rest-server/server/src/main/resources/reference.conf

```diff
@@ -10,3 +10,8 @@ rest-server {
   port = 9090
   ssl = false
 }
+
+auth {
+  enabled = false
```

Comment:
Cool.

When we create these compiled in setting we also have an environment variable to override them. So far all env is set or checked here:  https://github.com/actionml/harness/blob/master/rest-server/bin/harness-env